Spokestack/AppleSpeechRecognizer.swift

```diff
@@ -140,8 +140,8 @@ import Speech
                             a.confidence <= b.confidence }).first?.confidence ?? 0.0
                     strongSelf.context.transcript = r.bestTranscription.formattedString
                     strongSelf.context.confidence = confidence
-                    strongSelf.context.dispatch(.recognize)
                     strongSelf.vadFallWorker = DispatchWorkItem {[weak self] in
+                        self?.context.dispatch(.recognize)
```

Comment:
Putting this inside the vadFall task seems a little brittle to me. It seems better to either
a) check `r.isFinal` to determine whether to dispatch, or
b) set `shouldReportPartialResults` to `false` on the recognition request.

A note on b): we recently [started reporting partial results](https://github.com/spokestack/spokestack-android/pull/80) in the Android version.

Reply:
You'll love this: `isFinal` is never `true`. Been that way for years. Which is why `shouldReportPartialResults` must be `true`.

Reply:
Which is why this is all in my own custom vad fall timeout

Reply:
#$%@##%. Carry on.

Is `shouldReportPartialResults` similarly broken?

Reply:
Do you even have to ask? 😷

Reply:
to be fair, it's actually working and the only way to get a transcript, hence this other machinery to utilize it.